configure.ac

```diff
@@ -68,6 +68,10 @@ AC_CHECK_LIB([pthread], [pthread_create], [PTHREAD_LIBS=-lpthread
               AC_DEFINE([HAVE_LIBPTHREAD],,
                         [Whether libpthread was found])])
 
+# librt
+AC_CHECK_LIB([rt], [timer_create], [RT_LIBS=-lrt],
+             AC_MSG_ERROR("librt is required for performing timer-related operations"))
```

Comment:
It's not _universally_ required. This check should only be fatal where there is no `timer_create()` without linking in librt. If the test determines that `timer_create()` does not require linking with librt, this check should pass and `RT_LIBS` should be empty.